.env.example

```
@@ -2,3 +2,6 @@ ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY=
 ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY=
 ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT=
 DEFAULT_URL_HOST=
+APPLICATION_ID=
+PUBLIC_KEY=
+BOT_TOKEN=
```

If these are discord credentials, let's prefix them with something like `DISCORD_*`
```suggestion
DISCORD_APPLICATION_ID=
DISCORD_PUBLIC_KEY=
DISCORD_BOT_TOKEN=
```

Also can we start using rails credentials?
https://webcrunch.com/posts/the-complete-guide-to-ruby-on-rails-encrypted-credentials